build.cake

```diff
@@ -37,39 +37,48 @@ Task("Clean")
 // RESTORE PACKAGES
 //////////////////////////////////////////////////////////////////////
 
-Task("InitializeBuild")
+Task("NuGet-Package-Restore")
     .Does(() =>
     {
-      if (BuildSystem.IsRunningOnAppVeyor)
-      {
-          var tag = AppVeyor.Environment.Repository.Tag;
-
-          if (tag.IsTag)
-          {
-              packageVersion = tag.Name;
-          }
-          else
-          {
-              var buildNumber = AppVeyor.Environment.Build.Number;
-              packageVersion = version + "-CI-" + buildNumber + dbgSuffix;
-              if (AppVeyor.Environment.PullRequest.IsPullRequest)
-                  packageVersion += "-PR-" + AppVeyor.Environment.PullRequest.Number;
-              else if (AppVeyor.Environment.Repository.Branch.StartsWith("release", StringComparison.OrdinalIgnoreCase))
-                  packageVersion += "-PRE-" + buildNumber;
-              else
-                  packageVersion += "-" + AppVeyor.Environment.Repository.Branch;
-          }
-
-          AppVeyor.UpdateBuildVersion(packageVersion);
-      }
+        NuGetRestore(SOLUTION);
     });
 
 //////////////////////////////////////////////////////////////////////
+// INITIALIZE BUILD
+//////////////////////////////////////////////////////////////////////
+
+Setup(context =>
```

Comment:
Well, cake is a general scripting language and the Tasks could do anything. For example, we have targets in nunit that just run the tests, without building. That's why we created InitializeBuild. In fact, it doesn't matter much, because (1) everything is conditioned by IsRunningOnAppveyor and (2) there are no targets like that in this project. I think it's mostly a style thing - no need to change what you have done.